computer-vision/README.md

```diff
@@ -1,4 +1,34 @@
 # Computer Vision (OpenCV)
 
+## What is Computer Vision
+Computer Vision (CV) is a rapidly evolving field with applications in robotics, artificial intelligence and many other fields. For the purposes of a hackathon, computer vision projects typically include some type of real world video data (captured from a webcam). 
```

Comment:
Maybe talk about some of the other applications of vision in hackathons. This makes me think that all you can do with vision is videos. Maybe be like if you're interested in doing a project that has to do with finding something in a video or photo or grouping them together. Just to make it easier for ppl to realize what kinda tech they need for their projects

Reply:
I think it would be helpful to have the header kinda be a "if you're trying to do a project that does this this or this, this is the tech you should use"